projects/ontimize-web-ngx/src/lib/interfaces/o-table-oncellclick.interface.ts

```diff
@@ -0,0 +1,15 @@
+export interface OnCellClickTableEvent {
+
+  /** cell data */
+  cell: any;
+  /** column attr   */
+  columnId: string;
```

Comment:
We don't use the name `columnId` anywhere in the code. I think`column` or `columnName` are more suitable names for this parameter.

Reply:
I think `column` is not specific enough and can be ambiguous. If you prefer it can be used `columnName`